main.js

```diff
@@ -1,3 +1,4 @@
+/* Hamburger menu codes */
```

Comment:
- [Optional] Please try to make the navbar container on the mobile view take full width. so that it looks like the Figma design and professional. 

##### Your design 
![Screenshot from 2022-06-15 12-15-14](https://user-images.githubusercontent.com/58771507/173814575-d777444b-c476-4fc4-a6b1-3b5caab5f3dc.png)

##### Figma design
![Screenshot from 2022-06-15 12-15-58](https://user-images.githubusercontent.com/58771507/173814683-8bf9bb8a-af4b-4655-a1d0-9ecfc63d26fe.png)